README.md

```diff
@@ -15,23 +15,35 @@
 ### Run app
 On IDEA create new gradle configuration and use `bootRun` as the run argument with env variables
 
+| variable                   | value                            |
+|----------------------------|----------------------------------|
+| SPRING_DATASOURCE_USERNAME | user                             |
+| SPRING_DATASOURCE_PASSWORD | password                         |
+| SPRING_DATASOURCE_URL      | jdbc:postgresql://localhost/epoc |
+
+To run the application with security enabled add the following envs
```

Comment:
Maybe it's good to mention that jwt issuer and jwt jwk set uri:s can be found from secrets?